---
title: >
 STR Explorer - workflow for genome-wide association studies based on short tandem repeats
author:
  - name: Oxana (Sachenkova) Lundström
    affiliation: 1
    extra:
      - "oxana.lundstrom@dbb.su.se"
      - "&commat;merenlin"
  - name: Max Verbiest
    affiliation: 2
  - name: Maria Anisimova
    affiliation: 2
affiliation:
  - Stockholm University
  - Institute of Applied Simulations, ZHAW Wädenswil
date: "`r Sys.Date()`"
output:
  betterposter::poster_better:
    # Optional styling arguments
    hero_background: "#56B4E9"
    accent_color: "#005c99"
    hero_color: "#FFFFFF"
    logo: "https://yt3.ggpht.com/ytc/AAUvwnjGteYx8k6MAssnCXuvUKVwHYKOlapDEioYv5lGrg=s88-c-k-c0x00ffffff-no-rj"
    # See betterposter::qrcode_options() for help
    qrcode:
      text: "https://github.com/merenlin/mucin_poster"
      size: "250"
    mathjax: false
    # Generic rmarkdown arguments
    highlight: tango
    self_contained: true
references:
  - id: bychkovetal2018
    title: Deep learning based tissue analysis predicts outcome in colorectal cancer.
    author:
    - family: Bychkovetal
      given: D.
    container-title: Scientific Reports
    volume: 8
    URL: 'https://www.nature.com/articles/s41598-018-21758-3'
    DOI: 10.1038/s41598-018-21758-3 
    publisher:  Nature Publishing Group
    page: no. 1
    type: article-journal
    issued:
      year: 2018
  - id: tcga2012
    title: Comprehensive molecular characterization of human colon and rectal cancer
    author:
    - family: Cancer Genome Atlas 
      given: N.
    container-title: Nature 
    URL: 'https://doi.org/10.1038/nature11252'
    DOI: 10.1038/nature11252 
    issue: 487
    publisher: Nature Publishing Group
    page: 330–337
    type: article-journal
    issued:
      year: 2012
  - id: cptac2020
    title: Data from the Clinical Proteomic Tumor Analysis Consortium Colon Adenocarcinoma collection
    author:
    - family: National Cancer Institute Clinical Proteomic Tumor Analysis Consortium 
      given: (CPTAC)
    container-title: The Cancer Imaging Archive.
    URL: 'https://doi.org/10.7937/TCIA.YZWQ-ZZ63'
    DOI: 10.1038/nature11252 
    issued:
      year: 2020      

---

```{css echo=FALSE}
/* Adjust space at top of #hero (center panel) */
#hero {
  padding-top: 5%;
}
```


<div id="hero">
<!--
  Center Column of Poster

  * The main text goes in <h1> (single #) header.
  * You can add additional text if desired.
-->


# Pipeline to systematically search for **short tandem repeat (STR) biomarkers** in different medical conditions. Create your own workflow for personalised medicine research with STRs.

![](https://raw.githubusercontent.com/merenlin/mucin_poster/main/img/main_figure.png)

</div>



<div id="left-bar">
<!-- Text in the left bar, title and authors added automatically -->

# Why?

Microsatellite sequences are short tandem repeats(STRs) of two to six nucleotides. They are known to be hypervariable due to the accumulation of length mutations by intra-allelic polymerase slippage on microsatellite sequence during replication.

Current evidence suggests that STRs play an important role in cancer and other diseases. In colorectal cancer(CRC) variation in STR regions have been shown to influence protein-expression levels and increase tumor progression (Contente et al. 2002). The instability of microsatellite regions (MSI marker) is used by clinicians today to classify CRC tumors into different groups. This marker includes only five microsatellite regions and doesn’t exploit the whole variety of this phenomena.

Currently it is unknown whether distinct cancer groups exhibit distinct patterns of STR variation and in what CRC-related functional pathways STRs are involved.


# What?

STRs have been poorly studied until very recently due to their highly polymorphic nature which complicates their annotation. Moreover, genotyping and analyzing STR variants involves large amounts of data and working with many different biological data formats. 
We have developed a pipeline to systematically search for STR biomarkers in colorectal cancer that can be easily adapted for similar studies in other medical conditions. Our pipeline is the first building block towards a workflow for personalised medicine approach through STRs.
 
![](https://raw.githubusercontent.com/merenlin/mucin_poster/main/img/main_figure.png) [biomarkers and tcga data flowchart]


</div>



<div id="right-bar">
<!-- Right bar -->
# How? 

### STR genotyping workflow

The statistical framework TRAL  (Schaper 2015) have been used to find STRs in human reference genome. We share the logic behind creating a genotyping panel relevant for cancer research and how we arrived at the microsatellite panel we use today. These inferred STRs have been further genotyped on more than 400 genomes from patients with colorectal cancer available to us through the TCGA (The Cancer Genome Atlas Network 2012).

![](https://raw.githubusercontent.com/merenlin/mucin_poster/main/img/main_figure.png) [workflow illustration]

# Where?

### REST API

 The results are presented as a relational database with programmatic access through a REST-full API, - STR Explorer. Additional Python modules are available to add more genotyped data to the database .


STR Explorer is publically available at []. Comprehensive documentation of the API can be accessed on the main page. Fork the project on github. 

To access repeats of a list of genes of your interest, try the following: 

![](https://raw.githubusercontent.com/merenlin/mucin_poster/main/img/main_figure.png) [screenshot of usage]


# Future work

Next step is to combine STR annotations from STR Explorer with gene expression analysis and other clinically relevant data to perform a genome-wide association study (GWAS) on STRs in colorectal cancer as a case study. The inferred STR risk variants can later be validated as novel RNA and protein targets, which will serve as additional information for patient risk stratification and therapy-response prediction.

# References

</div>




 
