---
title: >
 STR Explorer - create your own workflow for personalized medical research with STRs.
author:
  - name: Oxana (Sachenkova) Lundström
    affiliation: 1
    extra:
      - "oxana.lundstrom@dbb.su.se"
      - "&commat;merenlin"
  - name: Max Verbiest
    affiliation: 2
  - name: Maria Anisimova
    affiliation: 2
affiliation:
  - Stockholm University
  - Institute of Applied Simulations, ZHAW Wädenswil
date: "`r Sys.Date()`"
output:
  betterposter::poster_better:
    # Optional styling arguments
    hero_background: "#457b9d"
    accent_color: "#1d3557"
    hero_color: "#FFFFFF"
    logo: ""
    # See betterposter::qrcode_options() for help
    qrcode:
      text: "https://github.com/merenlin/sibdays2022"
      size: "250"
    mathjax: false
    # Generic rmarkdown arguments
    highlight: tango
    self_contained: true
references:
  - id: guinney2015
    title: Deep learning based tissue analysis predicts outcome in colorectal cancer.
    author:
    - family: Guinney
      given: 
    container-title:  Nature Medicine
    volume: 21
    URL: 'https://www.nature.com/articles/nm.3967'
    DOI: 10.1038/nm.3967 
    publisher:  Nature Publishing Group
    page: no. 1350
    type: article-journal
    issued:
      year: 2015
  - id: mousavi2019
    title: Profiling the genome-wide landscape of tandem repeat expansions
    author:
    - family: Mousavi
      given: Nima
    container-title: Nucleic Acids Research
    volume: 47
    URL: 'https://doi.org/10.1093/nar/gkz501'
    DOI: 10.1093/nar/gkz501
    publisher:  Oxford Academic
    page: e90
    type: article-journal
    issued:
      year: 2019
  - id: tcga2012
    title: Comprehensive molecular characterization of human colon and rectal cancer
    author:
    - family: Cancer Genome Atlas 
      given: N.
    container-title: Nature 
    URL: 'https://doi.org/10.1038/nature11252'
    DOI: 10.1038/nature11252 
    issue: 487
    publisher: Nature Publishing Group
    page: 330–337
    type: article-journal
    issued:
      year: 2012
  - id: schaper2015
    title: TRAL, tandem repeat annotation library
    author: 
    - family: Schaper
      given: Elke
    container-title: Bioinformatics
    URL: 'https://doi.org/10.1093/bioinformatics/btv306'
    DOI: 10.1093/bioinformatics/btv306 
    issue: 18
    publisher: Oxford Academic
    page: 3051–3053
    type: article-journal
    issued:
      year: 2015
      
---

```{css echo=FALSE}
/* Adjust space at top of #hero (center panel) */
html {
  font-size: 32px;
}
#hero {
  padding-top: 5%;
}
```


<div id="hero">
<!--
  Center Column of Poster

  * The main text goes in <h1> (single #) header.
  * You can add additional text if desired.
-->


# Pipeline to systematically search for **short tandem repeat (STR) biomarkers** in different medical conditions. 

![](https://raw.githubusercontent.com/merenlin/sibdays2022/main/img/main_illustration.png)

### Fork the project on github to customize the workflow or use the current version at [**str-explorer.herokuapp.com**](str-explorer.herokuapp.com).

</div>



<div id="left-bar">
<!-- Text in the left bar, title and authors added automatically -->

# Why?

Microsatellite sequences are short tandem repeats(STRs) of two to six nucleotides. They are known to be hypervariable due to the accumulation of length mutations by intra-allelic polymerase slippage on microsatellite sequence during replication.

Current evidence suggests that STRs play an important role in cancer and other diseases. In colorectal cancer(CRC) variation in STR regions have been shown to influence protein-expression levels and increase tumor progression. The instability of microsatellite regions (**MSI marker**) is used by clinicians today to classify CRC tumors into different groups(@guinney2015). This marker includes only five microsatellite regions and doesn’t exploit the whole variety of this phenomena.

Currently it is unknown whether distinct cancer groups exhibit distinct patterns of STR variation and in what CRC-related functional pathways STRs are involved.


# What?

STRs have been poorly studied until very recently due to their highly polymorphic nature which complicates their annotation. Moreover, genotyping and analyzing STR variants involves large amounts of data and working with many different biological data formats. 
We have developed a pipeline to systematically search for STR biomarkers in **colorectal cancer** that can be easily adapted for similar studies in other medical conditions. Our pipeline is the first building block towards a workflow for **personalized medicine** approach through STRs.
 
![](https://raw.githubusercontent.com/merenlin/sibdays2022/main/img/uni_logos.png)
</div>



<div id="right-bar">
<!-- Right bar -->
# How? 

### STR genotyping workflow

The statistical framework **TRAL** (@schaper2015) have been used to find STRs in human reference genome. These inferred STRs have been further genotyped using **GangSTR** (@mousavi2019) on more than 400 genomes from patients with colorectal cancer available to us through the TCGA [@tcga2012]. Our pipeline then filters genotyped calls on sequencing quality, read coverage and ML estimate of GangSTR and transforms the vcf call files into records in our relational database. 

![](https://raw.githubusercontent.com/merenlin/sibdays2022/main/img/genotyping_flowchart.png)

# Where?

### REST API

 The results are presented as a relational database with programmatic access through a REST-full API, - STR Explorer. Additional Python modules are available to add more genotyped data to the database .
![](https://raw.githubusercontent.com/merenlin/sibdays2022/main/img/architecture_strexplorer.png)

Here is the example of usage maybe. 

```
This text is displayed verbatim / preformatted
This text is displayed verbatim / preformatted
This text is displayed verbatim / preformatted
This text is displayed verbatim / preformatted

```


</div>
# References




 
